Does not npm install

# I have node 4-LTS on Mac OS X

$ npm install --save aaronfi/chess-es6.js
npm ERR! addLocal Could not install /var/folders/27/j0p716dj6xxb3wqj1zf9_zwm0000gn/T/npm-12183-74d19e5d/git-cache-a45346b98a9a/9869b4d27db43aaa58e01c188419ea6e6d426062
npm ERR! addLocal Could not install /var/folders/27/j0p716dj6xxb3wqj1zf9_zwm0000gn/T/npm-12183-74d19e5d/git-cache-76cbad3ab981/9869b4d27db43aaa58e01c188419ea6e6d426062
npm ERR! addLocal Could not install /var/folders/27/j0p716dj6xxb3wqj1zf9_zwm0000gn/T/npm-12183-74d19e5d/git-cache-ba570ef71b6c/9869b4d27db43aaa58e01c188419ea6e6d426062
npm ERR! Darwin 15.5.0
npm ERR! argv "/usr/local/Cellar/node4-lts/4.4.5/bin/node" "/usr/local/bin/npm" "install" "--save" "aaronfi/chess-es6.js"
npm ERR! node v4.4.5
npm ERR! npm  v2.15.4

npm ERR! No version provided in package.json
npm ERR! 
npm ERR! If you need help, you may report this error at:
npm ERR!     <https://github.com/npm/npm/issues>

--- a/package.json
+++ b/package.json
@@ -1,5 +1,6 @@
 {
   "name": "engine",
+  "version": "0.1.0",
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1"
   },

